### ISSUE TITLE
Adhere to the Python naming convention

### ISSUE THREAD

**Issue by yakutovicha** [Probe-Particle/ppafm#93]
Here is a long and (I guess) complete guide to Python code style: [PEP8](https://peps.python.org/pep-0008/).

TL;DR

Just do that [1]:

Type | Public | Internal
-- | -- | --
Packages | lower_with_under |  
Modules | lower_with_under | _lower_with_under
Classes | CapWords | _CapWords
Exceptions | CapWords |  
Functions | lower_with_under() | _lower_with_under()
Global/Class Constants | CAPS_WITH_UNDER | _CAPS_WITH_UNDER
Global/Class Variables | lower_with_under | _lower_with_under
Instance Variables | lower_with_under | _lower_with_under
Method Names | lower_with_under() | _lower_with_under()
Function/Method Parameters | lower_with_under |  
Local Variables | lower_with_under

[1] https://namingconvention.org/python/
